resources/definitions/fdmprinter.def.json

```diff
@@ -4088,6 +4088,31 @@
                     "settable_per_extruder": true,
                     "limit_to_extruder": "adhesion_extruder_nr"
                 },
+                "z_offset_layer_0":
+                {
+                    "label": "Initial Layer Z Offset",
+                    "description": "The extruder is offset from the normal height of the first layer by this amount. It can be positive (raised) or negative (lowered). Some filament types adhere to the build plate better if the extruder is raised slightly.",
+                    "unit": "mm",
+                    "type": "float",
+                    "default_value": 0,
+                    "minimum_value": "-layer_height_0 * 0.5",
```

Comment:
@alekseisasin pointed out that it was possible for the user to specify values that would cause problems due to the inter-layer gaps being too small. So I tried to guard against that.